cincinnati/Cargo.toml

```diff
@@ -44,7 +44,6 @@ twoway = "^0.2"
 walkdir = "2.3.1"
 pretty_assertions = "0.6.1"
 test-case = "1.0.0"
-prettydiff = {}
```

Comment:
We need this there. See https://github.com/openshift/cincinnati/pull/232/commits/b27bc65effffb85c2951f6196ac7d7fd3db57999 for an actual fix.

Reply:
Oh, okay, that confused my local cargo build